README.md

```diff
@@ -10,49 +10,246 @@ It has only one function: generate the [Fibonacci sequence](https://en.wikipedia
+- `GPG_KEY_CONTENTS` should contain the contents of your GPG private key file, which you’ve created earlier in the [Export your private key](#export-your-private-key) section.
+
+#
+
+![](/images/image5.png)
+
+Note again that the names used for these secrets must match those used by the workflow that accesses their values.
+
+#### Create a release on GitHub
+
+With the workflow and secrets set up, you’re now ready to [create a release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) that will trigger the publication of your library.
+
+Go to your GitHub repository’s main page, and click on Releases in the menu in the right sidebar.
+
+![](/images/image3.png)
+
+Click *Draft a new release*.
+
+![](/images/image7.png)
+
+Each release creates a new tag. Set the name for the tag to be created, and set a name for the release (these may be identical). Note that setting a version here does not change the version of your coordinates configured in your `build.gradle.kts` file, so you should update that version before creating a new release.
+
+![](/images/image4.png)
+
+Double-check the branch you want to target with the release (especially if you want to release from a branch that’s different from your default), and add appropriate release notes for your new version.
+
+The checkboxes below allow you to mark a release as a pre-release (useful for alpha, beta, or RC versions of a library), or to set the release as the latest available one:
+
+![](/images/image6.png)
+
+Click the *Publish release* button to create the new release. This will immediately show up on your GitHub repository’s main page.
+
+Click the Actions tab on the top of your GitHub repository. Here you’ll see the new workflow that was triggered by the GitHub release. Click it to see the outputs of the publication task.
+
+After this task completes successfully, navigate to the [Deployments](https://central.sonatype.com/publishing/deployments) dashboard. You should see a new deployment here. This deployment will be in the *pending* and *validating* states for some time while Maven Central performs checks on it.
+
+Once your deployment moves to a *validated* state, you should see that it contains all the artifacts you’ve uploaded. If everything looks correct, click the *Publish* button to release these artifacts.
+
+![](/images/image2.png)
+
+Note that it will take some time (about 15-30 minutes, usually) after the release for the artifacts to be available publicly on Maven Central. Also note that the library may be available for use before the website is updated.
+
+There’s also another task available which both uploads and releases the artifacts automatically once the the deployment is verified, without having to manually release them on the website:
+
+| `./gradlew publishAndReleaseToMavenCentral` |
+| :---- |
+
+**Et voilà, you have successfully published your library to Maven Central.**
+
```

Comment:
What about having a 'Next steps' section?

E.g.

- Share your library with the Kotlin Community in the `#feed` channel in the [Kotlin Slack](https://kotlinlang.slack.com/) (To sign up visit https://kotl.in/slack)
- Add [shield.io badges](https://shields.io/badges/maven-central-version) to your README.